src/test/Drips.t.sol

```diff
@@ -106,6 +106,78 @@ contract DripsTest is DSTest {
         return recv(recv(recv1, recv2, recv3), recv4);
     }
 
+    function random(
+        bytes32 name,
+        uint256 salt,
+        uint256 range
+    ) internal pure returns (uint256) {
+        return uint256(keccak256(abi.encode(name, salt))) % range;
+    }
+
```

Comment:
Maybe something like that would suffice?

```solidity
bytes32 constant internal SEED_NOT_SET = "seed not set";
bytes32 seed = SEED_NOT_SET;

function random(uint256 range) internal returns (uint256) {
    require(seed != SEED_NOT_SET, "Seed not set"); 
    seed =  keccak256(bytes.concat(seed));
    return seed % range;
}

function testFuzzDripsReceiver(bytes32 fuzzSeed) public {
    seed = fuzzSeed;
    ...
}
```
The test run would be still easily reproducible, just feed the test with the same seed, but we would get rid of the awkward random numbers generation. It'd be also impossible to use `random` wrong, it'd always generate a fresh random number.

Reply:
Nice idea. I will add it. :+1: 

It will be a bit easier to use